examples/oapp/layerzero.config.ts

```diff
@@ -21,6 +21,10 @@ const config: OAppOmniGraphHardhat = {
     contracts: [
         {
             contract: fujiContract,
+            config: {
+                callerBpsCap: 1000, // this is example of how callerBpsCap can be configured when calling lz:oapp:wire
+                // QUESTION: is this accurate + should this be in edges or nodes?
+            },
```

Comment:
I will push tests in the next commit, but want to confirm if `callerBpsCap` should be specified at the node level or the edge level in the `connections` further down in this file. I placed it at the node level because the [pattern for configureOftBps](https://github.com/stargate-protocol/stargate-v2/blob/825d447ead52725cc128751db8d58cf823899865/packages/stg-evm-v2/devtools/src/oft-wrapper/config.ts#L16) configures it at the node level